<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageProcessingClass</class>
 <widget class="QMainWindow" name="ImageProcessingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageProcessing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="openImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开图片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pathChoosingButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>选择保存路径</string>
     </property>
    </widget>
    <widget class="QPushButton" name="convertToGrayscale">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>灰度图</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lowPassFilterButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>低通滤波</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="Binary">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>二值化</string>
     </property>
     <widget class="QRadioButton" name="toBinary">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>二值化</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSlider" name="thresholdSlider">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>128</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="thresholdSpinBox">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>128</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="highPassFilterButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>高通滤波</string>
     </property>
    </widget>
    <widget class="QPushButton" name="GaussianFilter">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>高斯滤波</string>
     </property>
    </widget>
    <widget class="QPushButton" name="edgeFilterButton">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>边缘滤波</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>0</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>颜色通道分离</string>
     </property>
     <widget class="QRadioButton" name="redShowButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>红</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="greenShowButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>绿</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="blueShowButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>蓝</string>
      </property>
     </widget>
     <widget class="QPushButton" name="splitButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>分离</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="imageFusionButton">
     <property name="geometry">
      <rect>
       <x>1050</x>
       <y>12</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>图像融合</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>59</y>
      <width>1331</width>
      <height>801</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="showOriFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="showOriLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="text">
       <string>                                                     显示</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="showFrame">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="showLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="text">
       <string>                                                     显示</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>641</width>
       <height>101</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="oriHistogramShowlabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>640</width>
        <height>90</height>
       </rect>
      </property>
      <property name="text">
       <string>                                                   直方图</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>490</y>
       <width>641</width>
       <height>101</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="histogramShowlabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>640</width>
        <height>90</height>
       </rect>
      </property>
      <property name="text">
       <string>                                                  直方图</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imageprocessing.qrc"/>
 </resources>
 <connections/>
</ui>
